modules-internal/alerting-helpers.bicep

```diff
@@ -0,0 +1,33 @@
+@description('The severity level of the alert')
+@allowed([
+  'Critical'
+  'Error'
+  'Warning'
+  'Informational'
+  'Verbose'
+])
+param severity string
+
+@description('The frequency at which the alert is evaluated, in minutes')
+param evaluationFrequencyMinutes int
+
+@description('The window size used to evaluate the alert, in minutes')
+param evaluationWindowSizeMinutes int = evaluationFrequencyMinutes
+
+
+var severityLookup = {
+  critical: 0
```

Comment:
Does is matter that these are a different case to the list above?

Reply:
Keys aren't case sensitive in bicep, but no harm in making them look consistent

Reply:
Updated